.idea/inspectionProfiles/IDE.xml

```diff
@@ -764,6 +764,7 @@
     <inspection_tool class="StaticCollection" enabled="true" level="INSPECTION" enabled_by_default="true">
       <option name="m_ignoreWeakCollections" value="false" />
     </inspection_tool>
+    <inspection_tool class="InterfaceMayBeAnnotatedFunctional" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
Here you turn on InterfaceMayBeAnnotatedFunctional, but alphabetically this looks like the position for SizeReplaceableByIsEmpty maybe? Is that one also turned on elsewhere?

Reply:
It has already been turned on in IDE.xml (line 742)

Reply:
<inspection_tool class="SizeReplaceableByIsEmpty" enabled="true" level="WARNING" enabled_by_default="true" />